### PR TITLE
[AzureDevOpsRelease] Support never deployed environments

### PR DESCRIPTION
**Pull request by Thilas**
So far, badges of never deployed environments were displayed as `invalid response data`. Here is a minor change to replace that by `never deployed`.

### CODE REVIEW

**Comment by Thilas** [comment]
@PyvesB, I noticed this issue while working on #4583. Would you mind to review as well please?

**Comment by Thilas** [comment]
Not sure to understand the reason of the test failure. They all pass locally...

File: services/azure-devops/azure-devops-release.tester.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ t.create('release status is succeeded')
     message: isBuildStatus,
   })
 
+t.create('never deployed')
+  .get('/totodem/8cf3ec0e-d0c2-4fcd-8206-ad204f254a96/1/4.json')
+  .expectBadge({
+    label: 'deployment',
+    message: 'never deployed',
```
Comment (PyvesB) [inline services/azure-devops/azure-devops-release.tester.js]: Are we confident that this will always be in a "never deployed" state?

Comment (Thilas) [reply services/azure-devops/azure-devops-release.tester.js]: We are as I setup this stage for that matter: https://dev.azure.com/totodem/Shields.io.

Comment (PyvesB) [reply services/azure-devops/azure-devops-release.tester.js]: Awesome!

**Comment by Thilas** [comment]
Thanks for reviewing so promptly! I'm looking forward having all these live.

**Comment by Thilas** [comment]
@PyvesB, unfortunately, tests are still failing.

**Comment by PyvesB** [comment]
> 
> 
> @PyvesB, unfortunately, tests are still failing.

Not anymore! 😃

**Pull request merged**